twisted/conch/ssh/keys.py

```diff
@@ -39,6 +42,47 @@
 from twisted.python.deprecate import deprecated, getDeprecationWarningString
 from twisted.python.versions import Version
 
+# Curve lookup table
+curveTable = {
```

Comment:
I added some @ivar declarations at the top of the module, but I don't know if it was done correctly.